Realm/RLMArrayLinkView.mm

```diff
@@ -375,7 +375,16 @@ - (RLMResults *)objectsWithPredicate:(NSPredicate *)predicate {
 - (NSUInteger)indexOfObjectWithPredicate:(NSPredicate *)predicate {
     auto query = translateErrors([&] { return _backingList.get_query(); });
     query.and_query(RLMPredicateToQuery(predicate, _objectInfo->rlmObjectSchema, _realm.schema, _realm.group));
+#if REALM_VER_MAJOR >= 2
```

Comment:
It's annoying to have to support 2 different versions - we should have updated master of core as well. If you want that there should not be any reason to not release a 1.x version with this change.